packages/scaffolding-cli/templates/src/ssr/server/index.ts

```diff
@@ -10,16 +10,22 @@ import helmetGuard from "./middlewares/helmet"
 import httpLogger from "./middlewares/http-logger"
 import cacheableResponse from "cacheable-response"
 import session from "express-session"
+
 import passport from "passport"
 import Auth0Strategy from "passport-auth0"
-import uid from "uid-safe"
+import conf from "../environment-configuration"
+import {setPassportSessionCookie} from "../lib/auth-cookies"
 
 let appInsights = AI
 if (!process.env.CI) {
     appInsights.setup().setAutoCollectConsole(true).start()
 }
 
 const port = parseInt(process.env.PORT || "3000", 10)
+const authenticationEnabled =
+    !!conf.AUTH0_CLIENT_SECRET && !!conf.AUTH0_CLIENT_ID
+const isRedisEnabled =
+    conf.REDIS_ENABLED && !process.env.CI && authenticationEnabled
```

Comment:
why not CI?

Reply:
Because we have not figured out how to handle auth in CI yet?

Reply:
until we attempt this [task](https://amido-dev.visualstudio.com/Amido-Stacks/_sprints/taskboard/Cycle%203%20-%20Front%20End%20-%20Part%202/Amido-Stacks/Cycle%203%20-%20Front%20End%20-%20Part%202/Sprint%207?workitem=1874)